integritybackend/config.py

```diff
@@ -65,8 +65,57 @@ def all_orgs(self):
         return self.config.keys()
 
     def get(self, org_id):
-        """Gets configuration dictionary for an org id."""
-        return self.config.get(org_id)
+        """Gets configuration dictionary for an org."""
+        if org_id in self.config:
+            return self.config.get(org_id)
+        else:
+            raise Exception(f"No organization with ID {org_id}")
+
+    def get_collections(self, org_id):
+        """Gets collection array for an org id."""
+        org_dict = self.get(org_id)
+        if "collections" in org_dict:
+            return org_dict.get("collections")
+        else:
+            raise Exception(f"No collections defined in {org_id}")
+
+    def get_collection(self, org_id, collection_id):
+        """Gets specific collection for an org."""
+        org_collections = self.get_collections(org_id)
+        collection_config = next(
+            (
+                c
+                for c in org_collections.values()
+                if c.get("conf").get("id") == collection_id
+            ),
+            None,
+        )
+        if collection_config is None:
+            raise Exception(f"No collection in {org_id} with ID {collection_id}")
+        return collection_config.get("conf")
+
+    def get_actions(self, org_id, collection_id):
+        """Gets action array for a collection."""
+        collection_conf = self.get_collection(org_id, collection_id)
+        org_dict = self.get(org_id)
+        if "actions" in collection_conf:
+            return collection_conf.get("actions")
+        else:
+            raise Exception(
+                f"No actions defined for {org_id}/{collection_id}"
+            )
+
+    def get_action(self, org_id, collection_id, action):
+        """Gets specific action for a collection."""
+        collection_conf = self.get_actions(org_id, collection_id)
+        action_config = next(
+            (c for c in collection_conf if c.get("name") == action), None
+        )
+        if action_config is None:
+            raise Exception(
+                f"No action in {org_id}/{collection_id} with action {action}"
+            )
+        return action_config.get("params")
```

Comment:
This is not returning the action, this is returning the action params. Either rename the func to `get_action_params` or change this.

Reply:
This has now been corrected and returns the action itself to mirror the behaviour of other get_ functions. 

`params` is now extracted in actions.py